lib/utils.pm

```diff
@@ -1084,6 +1084,7 @@ Also doing test ping to 10.0.2.2 to check that network is alive
 sub setup_static_network {
     my ($self, $ip) = @_;
     assert_script_run("echo 'default 10.0.2.2 - -' > /etc/sysconfig/network/routes");
+    assert_script_run("echo 'nameserver 10.160.0.1' >> /etc/resolv.conf");
```

Comment:
Can you explain this change in commit message? Otherwise LGTM.

Reply:
I'm sorry I've overlooked (thought you replace change in `/etc/sysconfig/network/routes` by change in `/etc/resolv.conf`). Sorry, no objections from my side.